### PR TITLE
[OKU-27]-FEAT- About Page

### PR DESCRIPTION
**Pull request by twentymurial33**
## Change Type
*Put an `x` in all the boxes that apply
- [] Bugfix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-29/about-page

### CODE REVIEW

File: project_app/components/AboutWrapper.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    display: block;
+    left: 380px;
+    top: 800px;
+    font-family: Quicksand-Regular;
+    font-size: 16px;
+    line-height: 30px;
+    color: #ffffff;
+  }
+`;
+
```
Comment (idiglove) [inline project_app/components/AboutWrapper.js:44]: include in a separate file

File: project_app/pages/About.tsx [hunk 2]
```diff
@@ -1,9 +1,60 @@
...
+    <HeaderWrapper>
+      <Header header={<NavbarText>About Us</NavbarText>}/> 
+      <HomePage/>
+    </HeaderWrapper>
+    <AboutWrapper/>
+   <FooterContainer>
+     <ProductContainer/>
+    <Footer/> 
+   </FooterContainer>
+    </AboutContainer>
```
Comment (idiglove) [inline project_app/pages/About.tsx]: run prettier on this pls

File: project_app/pages/About.tsx [hunk 3]
```diff
@@ -1,9 +1,60 @@
...
+text-align: center;
+
+ `
+
+const FooterContainer = styled.div`
+position:relative; 
+background-color: rgb(53, 53, 53);
+width:100%;
+`
+
```
Comment (idiglove) [inline project_app/pages/About.tsx:56]: include in a separate file

File: project_app/pages/About.tsx [hunk 4]
```diff
@@ -1,9 +1,60 @@
...
 
-function about() {
-  return (
-    <div>About</div>
-  )
-}
 
-export default about
\ No newline at end of file
+ function About() {
```
Comment (idiglove) [inline project_app/pages/About.tsx:11]: pls change filename to about.tsx

File: project_app/pages/Reviews.tsx [hunk 5]
```diff
@@ -73,7 +73,7 @@ export default function Reviews() {
       <ReviewBox>
          <button>Create a Review</button>
```
Comment (idiglove) [inline project_app/pages/Reviews.tsx:74]: pls chang filename to revews.tsx

**Review by idiglove — CHANGES_REQUESTED** [review]
just need some changes. I tested, and looks good

File: project_app/components/AboutWrapper.js (hunk 1 above)
Comment (twentymurial33) [reply project_app/components/AboutWrapper.js:44]: put in a separate file

File: project_app/pages/About.tsx (hunk 3 above)
Comment (twentymurial33) [reply project_app/pages/About.tsx:56]: Moved to style fie

File: project_app/pages/About.tsx (hunk 2 above)
Comment (twentymurial33) [reply project_app/pages/About.tsx]: Ran prettier

File: project_app/pages/About.tsx (hunk 4 above)
Comment (twentymurial33) [reply project_app/pages/About.tsx:11]: All filenames are lowercase 
Screen Shot 2023-05-23 at 4 57 52 PM

File: project_app/pages/Reviews.tsx (hunk 5 above)
Comment (twentymurial33) [reply project_app/pages/Reviews.tsx:74]: All filenames are lowercase 
Screen Shot 2023-05-23 at 4 57 52 PM

**Review by idiglove — APPROVED** [review]
good one

**Pull request merged**